content/correspondence/reference/API endpoints/initialize/_index.en.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Initialize
+linktitle: Initialize
+description: Endpoint for initializing a correspondence.
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence
+
+## Description
+This endpoint prepares and queues a correspondence for sending. Before using the endpoint and attachment must be uploaded by using the following endpoint in order to populate the "ExistingAttachments" field:
+
+/correspondence/api/v1/attachment/{{attachmentId}}/upload     
+
+(will add link here when doc is ready)
```

Comment:
To upload a attachment you first have to initialize, then upload. So multiple steps is required